packages/esbuild-plugin-pnp/sources/index.ts

```diff
@@ -12,19 +12,38 @@ async function defaultOnLoad(args: OnLoadArgs): Promise<OnLoadResult> {
   };
 }
 
-async function defaultOnResolve(args: OnResolveArgs, resolvedPath: string | null): Promise<OnResolveResult> {
+async function defaultOnResolve(args: OnResolveArgs, resolvedPath: string | null, error?: Error): Promise<OnResolveResult> {
```

Comment:
We should probably make these arguments an object at this point since https://github.com/yarnpkg/berry/pull/2919 also adds an argument